docs/data/joy/customization/right-to-left/RTLDemoJoy.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import rtlPlugin from 'stylis-plugin-rtl';
+import { prefixer } from 'stylis';
+import { CacheProvider } from '@emotion/react';
+import createCache from '@emotion/cache';
+import { CssVarsProvider, extendTheme } from '@mui/joy/styles';
+import FormControl from '@mui/joy/FormControl';
+import FormLabel from '@mui/joy/FormLabel';
+import FormHelperText from '@mui/joy/FormHelperText';
+import Input from '@mui/joy/Input';
+
+const theme = extendTheme({
+  direction: 'rtl',
+});
```

Comment:
@siriwatknp I'm guessing we'll need to enhance the `CssVarsProvider` component to accept the `direction` attribute? I'm getting an error with this!

<img width="500" alt="Screen Shot 2023-10-13 at 09 07 27" src="https://github.com/mui/material-ui/assets/67129314/93f60072-7674-4b4e-9d7a-e9e19570a87f">

Reply:
Got it, it's the theme options type.

Reply:
Thank you!